src/lib/hooks/useZAuctionCheck.ts

```diff
@@ -0,0 +1,23 @@
+import { useQuery } from 'react-query';
+
+import { useZnsSdk } from './useZnsSdk';
+
+export const useZAuctionCheck = (account: string, paymentToken: string) => {
+	const sdk = useZnsSdk();
+
+	return useQuery(
+		['check-zauction', { account, paymentToken }],
```

Comment:
`['user', 'approval', 'z-auction', { account, paymentToken }]` or similar

Reply:
updated in follow up pr